fatek/symbol.py

```diff
@@ -74,7 +74,8 @@ def __calculate_offset(self):
         offset = self.offset_dict[register]
         if register == 'C' and current_value and number >= 200:
             number %= 200
-            offset = 9700 + 2 * (number)
+            number = number + number
+            offset = 9700
```

Comment:
Hmm, I'm not really use counters in plc, so it may that its wrong. If you're sure about that, add that change